analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ParameterNameMatchesOriginal.cs

```diff
@@ -205,15 +205,4 @@ public class BasicImplementation : BaseInterface<int>
     {
         public void Apply(SomeInterface<int> intValueParam) { }   //Noncompliant FN
     }
-
-    // The following two classes reproduce: https://github.com/SonarSource/sonar-dotnet/issues/5005
-    public abstract class BaseRepro
-    {
-        public abstract int SomeMethod(int SomeParam1, int SomeParam2);
-    }
-
-    public class Reproducer : BaseRepro
-    {
-        public override int SomeMethod(int _, int SomeParam2) => SomeParam2; // Noncompliant FP
```

Comment:
What about keeping the test case and marking this as `// Noncompliant, it's not a discard`  to document the decision and prevent us from bumping it into it again in the future?